.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    # Weekly.
+    - cron: '0 0 * * 0'
+
+jobs:
+  exercises:
+    name: Check exercises
+    runs-on: ubuntu-22.04
+    container:
+      image: node:20
```

Comment:
Unless I'm mistaken, this requires the `node:20` docker image to be downloaded before the job runs. Do you really need node 20? If not, you could remove this as the `ubuntu-22.04` runner already contains node 18 installed: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Side note: it also has `jq` pre-installed, so you also don't need to install that.